### PR TITLE
add analytics

### PR DESCRIPTION
**Pull request by joannaskao**

### CODE REVIEW

File: src/js/audio.js [hunk 1]
```diff
@@ -1,10 +1,85 @@
...
+  fireEvent(ev.type, audio);
+}
+
+function shouldDispatch(progress) {
+  const relevantProgressPoints = [8, 9, 10, 11, 12,
+                                  23, 24, 25, 26, 27,
+                                  48, 49, 50, 51, 52,
+                                  73, 74, 75, 76, 77,
+                                  100];
+  return relevantProgressPoints.includes(progress);
```
Comment (tom-parker) [inline src/js/audio.js:18]: Is there a chance multiple tracking events will get fired around the quartiles, e.g we could see events fired at 24/25/26%?

**Comment by tom-parker** [comment]
Looks good to be bar the comment

File: src/js/audio.js (hunk 1 above)
Comment (joannaskao) [reply src/js/audio.js:18]: That will probably happen. I think o-video does the same thing at the moment. https://github.com/Financial-Times/o-video/blob/master/src/js/video.js#L48. I can probably write something on top that filters out extraneous events when there are multiple progress events per quartile. Would that make sense to do?

Comment (tom-parker) [reply src/js/audio.js:18]: If it's aligned with o-video then I'd keep it the same for now but we might need to review that at some point in the future

Comment (joannaskao) [reply src/js/audio.js:18]: Sounds good! I'll look out for it — if it gets changed in the future in o-video, let me know! Thanks!

**Pull request merged**